apps/edxapp/templates/services/cms/job_01_create_directories.yml.j2

```diff
@@ -0,0 +1,47 @@
+{%- from "apps/edxapp/templates/services/cms/macros/image.yml.j2" import image_name -%}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: edxapp-cms-create-directories-{{ job_stamp }}
+  namespace: "{{ project_name }}"
+  labels:
+    app: edxapp
+    service: cms
+    version: "{{ edxapp_image_tag }}"
+    job_stamp: "{{ job_stamp }}"
+    # Jobs with the "pre" job type will be executed prior to deployments
+    job_type: "pre"
+    deployment_stamp: "{{ deployment_stamp }}"
+spec:
+  template:
+    metadata:
+      name: edxapp-cms-create-directories-{{ job_stamp }}
+      labels:
+        app: edxapp
+        service: cms
+        version: "{{ edxapp_image_tag }}"
+        job_stamp: "{{ job_stamp }}"
+        deployment_stamp: "{{ deployment_stamp }}"
+    spec:
+      containers:
+        - name: edxapp-cms-create-directories-{{ job_stamp }}
+          # We point to a local registry image build for this "live" image (see
+          # ImageStream and BuildConfig templates)
+          image: "{{ internal_docker_registry }}/{{ project_name }}/{{ image_name('cms', edxapp_image_tag, edxapp_theme_tag) }}"
+          command:
+            - "bash"
+            - "-c"
+            - {% for path in required_directories %}(mkdir {{ path }} || true) && {% endfor %}echo "✅"
```

Comment:
You should use `-p` in case of nested, deep paths and it always return a success exit code.
```suggestion
            - {% for path in required_directories %}mkdir -p {{ path }} && {% endfor %}echo "✅"
```

Reply:
:medal_sports: